confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java

```diff
@@ -432,14 +433,18 @@ public void startHeartbeatService() {
 
   /** loop body of the heartbeat thread */
   private void heartbeatLoopBody() {
-    if (getConsensusManager().isLeader()) {
-      // Generate HeartbeatReq
-      THeartbeatReq heartbeatReq = genHeartbeatReq();
-      // Send heartbeat requests to all the registered DataNodes
-      pingRegisteredDataNodes(heartbeatReq, getRegisteredDataNodes());
-      // Send heartbeat requests to all the registered ConfigNodes
-      pingRegisteredConfigNodes(heartbeatReq, getRegisteredConfigNodes());
-    }
+    Optional.ofNullable(getConsensusManager())
+        .ifPresent(
```

Comment:
We'd better add annotation about why should we must use Optional.ofNullable.ifPresent here.

Reply:
Done